nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/csv/JacksonCSVRecordReader.java

```diff
@@ -52,11 +52,12 @@ public class JacksonCSVRecordReader extends AbstractCSVRecordReader {
     private final MappingIterator<String[]> recordStream;
     private List<String> rawFieldNames = null;
     private boolean allowDuplicateHeaderNames;
+    private final boolean trimDoubleQuote;
 
     private volatile static CsvMapper mapper = new CsvMapper().enable(CsvParser.Feature.WRAP_AS_ARRAY);
 
     public JacksonCSVRecordReader(final InputStream in, final ComponentLog logger, final RecordSchema schema, final CSVFormat csvFormat, final boolean hasHeader, final boolean ignoreHeader,
```

Comment:
Same as in the case of CSVRecordReader: I recommend reintroducing the declaration of the original constructor, and the implementation should just call this constructor with trimDoubleQuote=true. This way everywhere where you called this constructor with trimDoubleQuote=true, you could revert to calling the original constructor.